### PR TITLE
logo upload & readme update

### PR DESCRIPTION
**Pull request by Tobaloidee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="/logo/logotype-horizontal.png"></p>
```
Comment (tido64) [inline README.md:1]: Could you add an `alt` attribute with an appropriate string, e.g. `Logo courtesy of @Tobaloidee` or with your preferred name. It is only right that we give you credit for this.

Comment (tido64) [inline README.md:1]: Could you also move the images under `src/Resources/`?

**Review by tido64 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tido64) [reply README.md:1]: You can also use the Markdown syntax here, i.e. `Logo by Tobaloidee`

**Comment by Tobaloidee** [comment]
it's already updated Sir...I asked my assistant to edit it. Thanks!

**Comment by tido64** [comment]
Merged #14.

**Pull request closed without merging**